src/components/strategies/overlapping/OverlappingStrategyGraph.tsx

```diff
@@ -518,8 +520,14 @@ export const OverlappingStrategyGraph: FC<Props> = (props) => {
   return (
     <figure className="relative">
       <figcaption className="text-10 absolute inset-x-0 top-0 flex items-center justify-center gap-4 p-16 text-white/60">
-        <span>Market price provided by CoinGecko</span>
-        <IconCoinGecko className="size-8" />
+        {props.externalPrice === props.marketPrice ? (
```

Comment:
```suggestion
        {props.externalPrice === +props.marketPrice ? (
```

To prevent showing "User-defined market price" in deposit and withdraw flows as the DisableProps use typeof marketPrice = string